tw-tasks-core/src/main/java/com/transferwise/tasks/helpers/ICoreMetricsTemplate.java

```diff
@@ -49,7 +51,7 @@ public interface ICoreMetricsTemplate {
 
   void registerFailedNextEventTimeChange(String taskType, ZonedDateTime fromNextEventTime, ZonedDateTime toNextEventTime);
 
-  void registerTaskGrabbingResponse(String bucketId, String type, int priority, ProcessTaskResponse processTaskResponse);
+  void registerTaskGrabbingResponse(String bucketId, String type, int priority, long epochMilliBeforeProcessing, ProcessTaskResponse processTaskResponse, TaskTriggering taskTriggering);
```

Comment:
Would rather not expose `TaskTriggering` in this interface, is there a better way 🤔